ckan/logic/action/get.py

```diff
@@ -323,7 +323,9 @@ def _group_or_org_list(context, data_dict, is_org=False):
         if errors:
             raise ValidationError(errors)
     sort = data_dict.get('sort') or config.get_value('ckan.default_group_sort')
-    q = data_dict.get('q')
+    k = data_dict.get('q')
+    q = k.strip()
```

Comment:
This will fail if `q` is None (that's why you get the test failures)

Reply:
@amercader thankyou for reviewing my pr. I did some changes in my code and it will work when q is None.